liquidctl/driver/coolit.py

```diff
@@ -0,0 +1,420 @@
+"""liquidctl driver for Corsair Platinum and PRO XT coolers.
+
+Supported devices
+-----------------
+
+ - Corsair H110i GT
+
+Supported features
+------------------
+
+ - general monitoring
+ - pump speed control
+ - fan speed control
+
+Copyright (C) 2020–2020  Jonas Malaco
```

Comment:
Perhaps a stupid question: is it necessary to put 2020 twice?

Reply:
You're right, this is useless if contribution was over a single year.

Reply:
We no longer include dates in the copyright notices, with only a few exceptions.

```
Copyright Roberto Marques, Serphentas  and contributors
SPDX-License-Identifier: GPL-3.0-or-later
```

Related: 1cc8cf03d78bb269f8614e42aadcae6a1acb350f
Also: [process.md#license-and-copyright](https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md#license-and-copyright)

---

I also don't know why I'm listed as the first co-author here... I don't remember having worked on it directly (although this driver has been in the works for a while, so I might just have forgotten about it).

Was there something from me in the old PR?

Reply:
I think this is an old-style attribution per old PR (see [files](https://github.com/liquidctl/liquidctl/pull/147/files)). I didn't find any commit by you (@jonasmalacofilho) so I'll just roll with myself and @RobertoBiundo.